versions/src/main/java/cc/polyfrost/oneconfig/internal/OneConfig.java

```diff
@@ -157,6 +146,58 @@ public static void init() {
         initialized = true;
     }
 
+    //#if FORGE==1 && MC<=11202
+
+    private static void handleForgeCommand(ModContainer mod) {
+        for (Mod configMod : ConfigCore.mods) {
+            final String configModName = configMod.name.toLowerCase(Locale.ENGLISH);
+            if (Objects.equals(configModName, mod.getName()) || Objects.equals(configModName, mod.getModId())) {
+                return;
+            }
+        }
+        for (net.minecraft.command.ICommand command :  net.minecraftforge.client.ClientCommandHandler.instance.getCommands().values()) {
+            if (Objects.equals(command.getCommandName(), mod.getModId())) {
+                ForgeCompat.compatMods.put(new ForgeCompat.ForgeCompatMod(mod.getName(), ModType.THIRD_PARTY), () -> new TickDelay(() -> {
+                    UScreen.displayScreen(null);
+                    try {
+                        //execute
+                        //#if MC<=10809
+                        command.processCommand(UPlayer.getPlayer(), new String[]{});
```

Comment:
Why not passing any args to the command?

Reply:
because usually that opens the config